base/base/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ if (USE_CLANG_TIDY)
     set (CMAKE_CXX_CLANG_TIDY "${CLANG_TIDY_PATH}")
 endif ()
 
+# TODO: Remove this. We generally like to compile with C++23 but Clang currently crashes with our libcxx
```

Comment:
Did we report it to clang? And what about clang-16?

Reply:
Will try clang 16 once it's released. I just know that @amosbird recently did some fixes in preparation for clang 16 (#47027).

I would assume that just a newer compiler won't help. We'd likely also need to upgrade libcxx and libcxxabi (+ our internal LLVM).

Perhaps it's better then to defer this PR by a little bit and make it dependent on Clang 16.